index.html

```diff
@@ -3,8 +3,7 @@
 
   <head>
     <meta charset="utf-8">
-    <link rel="stylesheet" type="text/css" href="css/main.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
```

Comment:
@LaurenAMolloy - we want to keep these two lines here as they are - remember that this is the main index.html file - where all student links are. Where you want your own css file to be included is in your own advice file (which you have done - so we're all good there!)